actionpack/lib/action_dispatch/system_testing/browser.rb

```diff
@@ -7,6 +7,7 @@ class Browser # :nodoc:
 
       def initialize(name)
         @name = name
+        @option = nil
```

Comment:
any reason to initialize this ivar ?

Reply:
yes, a ruby warning about an uninitialized ivar when running the tests -.-